x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -140,6 +136,18 @@ export const signalRulesAlertType = ({
         await ruleStatusService.error(gapMessage, { gap: gapString });
       }
       try {
+        const { listClient, exceptionsClient } = await getListsClient({
```

Comment:
NOTE: Now needing to load exception items, where as before they lived on the rule.